TASVideos/Pages/Shared/_NavBarPartial.cshtml

```diff
@@ -1,31 +1,31 @@
-<navbar>
-	<nav-dropdown activate="Home">
```

Comment:
bootstrap font-weight-bold class should be used in place of the b element

Reply:
Fixed with `fw-bold`.